iff -Naur a/src/gui/DatabaseWidget.cpp b/src/gui/DatabaseWidget.cpp
--- a/src/gui/DatabaseWidget.cpp	2018-11-23 07:07:32.075892080 +0000
+++ b/src/gui/DatabaseWidget.cpp	2018-11-23 10:41:03.811825940 +0000
@@ -27,6 +27,7 @@
 #include <QSplitter>
 #include <QTimer>
 #include <QProcess>
+#include <QHeaderView>
 
 #include "autotype/AutoType.h"
 #include "core/Config.h"
diff -Naur a/src/gui/entry/EditEntryWidget.cpp b/src/gui/entry/EditEntryWidget.cpp
--- a/src/gui/entry/EditEntryWidget.cpp	2018-11-23 07:07:32.084892079 +0000
+++ b/src/gui/entry/EditEntryWidget.cpp	2018-11-23 10:41:03.828825939 +0000
@@ -21,6 +21,7 @@
 #include "ui_EditEntryWidgetHistory.h"
 #include "ui_EditEntryWidgetMain.h"
 
+#include <QButtonGroup>
 #include <QDesktopServices>
 #include <QStackedLayout>
 #include <QStandardPaths>
diff -Naur a/src/gui/entry/EntryView.h b/src/gui/entry/EntryView.h
--- a/src/gui/entry/EntryView.h	2018-11-23 07:07:32.087892079 +0000
+++ b/src/gui/entry/EntryView.h	2018-11-23 10:41:03.830825939 +0000
@@ -18,6 +18,7 @@
 #ifndef KEEPASSX_ENTRYVIEW_H
 #define KEEPASSX_ENTRYVIEW_H
 
+#include <QHeaderView>
 #include <QTreeView>
 
 #include "gui/entry/EntryModel.h"
